app/src/main/java/org/thoughtcrime/securesms/database/Storage.kt

```diff
@@ -1481,21 +1548,20 @@ open class Storage(
             || (userPublicKey == recipientPublicKey && userPublicKey == senderPublicKey)
         ) return
 
-        val recipientDb = DatabaseComponent.get(context).recipientDatabase()
-        val threadDB = DatabaseComponent.get(context).threadDatabase()
+        val recipientDb = recipientDatabase
+        val threadDB = threadDatabase
```

Comment:
It doesn't look like these two new vals are needed since they match existing properties (newly injected)